server/schemas/resolvers.js

```diff
@@ -1,6 +1,7 @@
 const { User, Book, Order } = require('../models');
 const { signToken, AuthenticationError } = require('../utils/auth');
 const bcrypt = require("bcrypt");
+require('dotenv').config();
```

Comment:
Minor detail but this line really should be at the very top of the server.js file.

Reply:
ill remove it from there, it's not needed. I don't even remember putting it in that file